tests/test_match_counter.py

```diff
@@ -59,64 +59,6 @@ def test_matched(self):
         self.assertCountEqual(counter.matched_keys(pkg), ['url', 'summary'])
         self.assertCountEqual(counter.matched_haystacks(pkg), [url, summary])
 
-    @mock.patch('dnf.match_counter.MatchCounter._eval_distance', return_value=0)
-    def test_sorted(self, _):
-        counter = dnf.match_counter.MatchCounter()
-        self.assertEqual(counter.sorted(), [])
-
-        counter = dnf.match_counter.MatchCounter()
-        pkg1, pkg2, pkg3 = PackageStub().several(3)
-        counter.add(pkg1, 'name', '')
-        counter.add(pkg2, 'summary', '')
-        self.assertEqual(counter.sorted(), [pkg2, pkg1])
-
-        counter.add(pkg3, 'url', '')
-        self.assertEqual(counter.sorted(), [pkg3, pkg2, pkg1])
-        self.assertEqual(counter.sorted(reverse=True), [pkg1, pkg2, pkg3])
-
-    @mock.patch('dnf.match_counter.MatchCounter._eval_distance', return_value=0)
-    def test_sorted_with_needles(self, _):
-        # the same needles should be listed together:
-        counter = dnf.match_counter.MatchCounter()
-        pkg1, pkg2, pkg3, pkg4 = PackageStub().several(4)
-        counter.add(pkg1, 'summary', 'grin')
-        counter.add(pkg2, 'summary', 'foolish')
-        counter.add(pkg3, 'summary', 'grin')
-        counter.add(pkg4, 'summary', 'grin')
-
-        srt = counter.sorted()
-        self.assertIn(srt.index(pkg2), (0, 3))
-
-        # more unique needles is more than less unique needles:
-        counter = dnf.match_counter.MatchCounter()
-        counter.add(pkg1, 'summary', 'a')
-        counter.add(pkg1, 'summary', 'b')
-        counter.add(pkg2, 'summary', 'b')
-        counter.add(pkg2, 'summary', 'b')
-
-        self.assertSequenceEqual(counter.sorted(), (pkg2, pkg1))
-
-    @mock.patch('dnf.match_counter.MatchCounter._eval_distance', return_value=0)
-    def test_sorted_limit(self, _):
-        counter = dnf.match_counter.MatchCounter()
-        pkg1, pkg2, pkg3 = PackageStub().several(3)
-        counter.add(pkg1, 'name', '')
-        counter.add(pkg2, 'url', '')
-        counter.add(pkg3, 'description', '')
-        self.assertSequenceEqual(counter.sorted(limit_to=[pkg1, pkg2]),
-                                 (pkg2, pkg1))
-
-    @mock.patch('dnf.match_counter.MatchCounter._eval_distance', return_value=0)
-    def test_sorted_exact_match(self, _):
-        """Exactly matching the name beats name and summary non-exact match."""
-        counter = dnf.match_counter.MatchCounter()
-        pkg1 = PackageStub('wednesday-1-1.noarch', 'morning')
-        pkg2 = PackageStub('wednesdaymorning-1-1.noarch', "5 o'clock")
-        counter.add(pkg1, 'name', 'wednesday')
-        counter.add(pkg2, 'name', 'wednesday')
-        counter.add(pkg2, 'summary', 'clock')
-        self.assertSequenceEqual(counter.sorted(), (pkg2, pkg1))
```

Comment:
I think it's a bit of a waste to delete these tests? You cna probably simplify them quite a bit, but leaving a couple tests for the actual ordering would be useful? I know you've added the tests to the CI stack, but unit tests are also good to have... (the `_eval_distance` method was overriden to return 0 anyway, so that makes it easier too)